.github/pull_request_template.md

```diff
@@ -19,6 +19,16 @@ Bad example:
 change ingress
 -->
 
+### Risk Assessment
```

Comment:
```suggestion
### Risk Assessment ([more details](https://app.getguru.com/card/iMnRRRjT/PR-Risk-Assessment))
```

I'm not sure we want to add the link in the header, though (i.e. will it look weird?)

Reply:
This look okay? 
![image](https://github.com/customink/.github/assets/13353469/0bbf08bc-dde3-47fc-81ba-51bf8be435e7)

Reply:
💯